docs/rfcs/1-add-pair-for-reacher.md

```diff
@@ -0,0 +1,64 @@
+- Author: zu1k <i@lgf.im>
+- Start Date: 2021-07-15
+- RFC PR: [beyondstorage/go-service-ipfs#13](https://github.com/beyondstorage/go-service-ipfs/pull/13)
+- Tracking Issue: [beyondstorage/go-service-ipfs#14](https://github.com/beyondstorage/go-service-ipfs/issues/14)
+
+# RFC-1: Add a pair for `Reacher`
```

Comment:
How about using PR numbers as the RFC ID?

- Following other beyondstorage's project style
- Easy to identify
- Avoid conflict

Reply:
Maybe we can make the RFC more readable by changing the name into `Add Gateway Pair`?